package.json

```diff
@@ -14,9 +14,9 @@
     "asar": "^0.13.0",
     "bower": "^1.3.12",
     "electron": "^1.6.1",
-    "electron-builder": "^17.0.1",
+    "electron-builder": "^18.8.1",
     "electron-icon-maker": "^0.0.3",
-    "electron-publisher-s3": "^17.0.2",
+    "electron-publisher-s3": "^19.0.1",
```

Comment:
Here are the changes that I could find from the mega-changelog that seemed relevant:

# electron-builder 17.0.1 -> 18.8.1

## Bug fixes:

* deployment: s3 publisher md5 integrity (b57dc8a)
* deployment: latest.yml is completely empty when uploaded to S3 bucket (4b25ca2), closes #1582
* "status 401: Unauthorized" issue with dl.bintray.com (52995df), closes #1581
* electron-builder 17.5.0, 17.8.0: cannot find module 'debug' (5835654), closes #1564

## Features:

* grab latest electron version from github if not specified (a826df2)
* linux: add missing Exec variable for passing URLs as arguments (4a87e67), closes #1592
* local path to custom electron build (windows support) (521aea6), closes #1534 #1342

# electron-updater 1.11.2 -> 2.1.2

## Breaking changes:

* updater: remove compatibility with very old electron-updater version (< 0.10.0)

## Bug fixes:

* check for EACCES error when try to install on auto updated windows (#1636) 
* isSilent is optional (12473d0)
* MacUpdater — close proxy server after download (39ae0a4)
* electron-updater v1.14.* throws cannot find module 'debug' (78d9b33), closes #1521
* Autoupdates to lower version with allowPrerelease=true using GitHub releases (36fc3db), closes #1497
* Upload release failed. Response status: 401 Unauthorized (257a7dd), closes #1385
* electron-updater: Incorrect comparison of version numbers (electron-updater) (17ac619), closes #1488

## Features
* ensure that update only to the application signed with same cert (66771d3), closes #1187
* Full support of any existing electron-updater features on macOS.
 * Progress is supported.
 * Any dynamic or relative URL is supported.
 * Any provider as you want without any restrictions. Yes — private github repo is supported (#1370). Yes — Bintray is supported now (#1172).
* autoUpdater download-progress event is not called on macOS (a75bac8), closes #1167
* prepare to support private github repo, nuts and so on on macOS (a41936b), closes #1575 #1571 #1577
* isSilent param of quitAndInstall method #1545 (daeefa6)
* add proxy authentication support to electron-updater (a892a5b), closes #1530
* ${buildVersion} macro in artifactName config (23f0b37), closes #1527
* mac: allow build for pull requests / code sign artifacts (9dbc789), closes #1524
* GitHub: Allow pre-release builds to be auto updated (f275831), closes #1391
* support additional certificate file (#1467) (19c8ee4)